kafka_consumer/README.md

```diff
@@ -1,32 +1,54 @@
-# Kafka_consumer Integration
+# kafka_consumer Integration
 
-## Overview
+# Overview
 
-Get metrics from kafka_consumer service in real time to:
+This Agent check only collects metrics for Kafka broker offset, consumer offset, and consumer lag. If you want to collect many more Kafka metrics, see the kafka check.
 
-* Visualize and monitor kafka_consumer states
-* Be notified about kafka_consumer failovers and events.
+# Installation
 
-## Installation
+The Agent's Kafka consumer check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your Kafka nodes. If you need the newest version of the check, install the `dd-check-kafka-consumer` package.
 
-Install the `dd-check-kafka_consumer` package manually or with your favorite configuration manager
+# Configuration
 
-## Configuration
+Create a `kafka.yaml` in the Datadog Agent's `conf.d` directory:
 
-Edit the `kafka_consumer.yaml` file to point to your server and port, set the masters to monitor
+```
+init_config:
+#  Customize the ZooKeeper connection timeout here
+#  zk_timeout: 5
+#  Customize the Kafka connection timeout here
+#  kafka_timeout: 5
 
-## Validation
+instances:
+  # - kafka_connect_str: localhost:9092
```

Comment:
These options, I believe, are not currently optional, so I don't think they should be commented out:
- `kafka_connect_str`
- `zk_connect_str`

`consumer_groups` are now "discoverable", but with the caveat of hitting zookeeper pretty hard, so we might want to add a note with regard to that. I believe the updated YAML (we merged an update recently) already does, so I'll let you decide if it's necessary here or not (you might want to give it a look anyways).